app/validators/http_url_validator.rb

```diff
@@ -0,0 +1,11 @@
+class HttpUrlValidator < ActiveModel::EachValidator
+  include UrlValidation
+
+  def validate_each(record, attribute, value)
+    return if options[:allow_blank] && value.empty?
```

Comment:
This doesn't seem to make sense in a generic validator?

Reply:
every validator should implement `:allow_blank` if possible, otherwise `validate :attribute, http_url: true, allow_blank: true` will not work as expected, or will it?

Reply:
@tbrisker anything else to add? the code looks good to me